package.json

```diff
@@ -107,7 +107,7 @@
     "ms": "0.7.1",
     "name-all-modules-plugin": "1.0.1",
     "node-sass": "3.7.0",
-    "notifications-panel": "1.2.7",
+    "notifications-panel": "git+https://github.com/Automattic/notifications-panel#react16",
```

Comment:
should this really be the secure link? will people be able to download it without credentials?

Reply:
If it doesn't we can publish a 2.0.0 tomorrow.

Reply:
I saw the same thing on line 115: https://github.com/Automattic/wp-calypso/pull/19083/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R115

I'll change it to be an insecure link

Reply:
🤷‍♂️